source/team-guide/modifying-scps.html.md.erb

```diff
@@ -0,0 +1,19 @@
+---
+title: Modifying Service Control Policies (SCPs)
+---
+
+# <%= current_page.data.title %>
+
+We use the following SCPs for the Modernisation Platform to restrict certain actions at an account level.
+
+SCP | Location | Description
+---|---|---|
+Modernisation Platform Core | [`terraform/environments/scp_mp_core`](https://github.com/ministryofjustice/modernisation-platform/blob/main/terraform/environments/scp_mp_core.tf) | SCP for accounts with the account type of "core"
+Modernisation Platform Member | [`terraform/environments/scp_mp_member`](https://github.com/ministryofjustice/modernisation-platform/blob/main/terraform/environments/scp_mp_member.tf) | SCP for accounts with the account type of "member"
+Modernisation Platform Core | [`terraform/environments/scp_mp_core`](https://github.com/ministryofjustice/modernisation-platform/blob/main/terraform/environments/scp_mp_member_unrestriced.tf) | SCP for accounts with the account type of "member-unrestricted"
```

Comment:
Should this line start with 'Modernisation Platform Member Unrestricted'

Reply:
Good spot thanks